server/resolvers/login.go

```diff
@@ -141,7 +145,18 @@ func LoginResolver(ctx context.Context, params model.LoginInput) (*model.AuthRes
 		log.Debug("MFA service not enabled: ", err)
 	}
 
-	if refs.BoolValue(user.IsMultiFactorAuthEnabled) && !isMFADisabled {
+	isTOTPLoginDisabled, err := memorystore.Provider.GetBoolStoreEnvVariable(constants.EnvKeyDisableTOTPLogin)
```

Comment:
@lemonScaletech can we document these scenarios in docs under login API

thanks

Reply:
@lakhansamani I have already did it in https://github.com/authorizerdev/docs/pull/39/files
do we need something else ?

thanks

Reply:
Lets add conditions for enable for MFA and all in note there.

Reply:
okay let me do it and revert back to you.

Reply:
https://github.com/authorizerdev/docs/pull/49

I have created a PR in docs, does this fulfil our needs ?